pss/pss.go

```diff
@@ -185,7 +189,10 @@ func (o outbox) msg(slot int) *outboxMsg {
 }
 
 func (o outbox) free(slot int) {
-	o.slots <- slot
+	select {
```

Comment:
the old code was blocking when outbox is closed, potentially blocking and leaking processOutbox internal go routine